test/Elastica/Aggregation/GeoBoundsTest.php

```diff
@@ -8,7 +8,7 @@
 
 class GeoBoundsTest extends BaseAggregationTest
 {
-    protected function _getIndexForTest()
+    private function getIndexForTest()
```

Comment:
That is an interesting discussion we should probably have. I was stubborn enough to keep Elastica with `_` for all private and protected function as I personally liked it. But I think in the meantime, most projects moved away from it?